components/robotsports/getball-intercept/test.cpp

```diff
@@ -0,0 +1,78 @@
+// this file was produced by MRA-codegen.py from template_test.cpp
+// with the intent of allowing user to add custom tests
+
+
+// Include testframework
+#include "gtest/gtest.h"
+#include "gmock/gmock.h"
+#include "test_factory.hpp"
+using namespace ::testing;
+#include <cmath>
+
+// System under test:
+#include "RobotsportsGetballIntercept.hpp"
+using namespace MRA;
+
+// Basic tick shall run OK and return error_value 0.
+TEST(RobotsportsGetballInterceptTest, basicTick)
```

Comment:
test name does not match content. `basicTick` is an auto-generated testcase, I would suggest to restore it and rename this specific test.

Reply:
renamed the testcase and restored the basicTick